source/components/atoms/Input/Input.tsx

```diff
@@ -77,7 +77,7 @@ const StyledAccessoryViewChild = styled(View)`
   paddingHorizontal: 8px;
 `;
 
-const _replaceSpace = str => (str?.replace ? str.replace(/\u0020/, '\u00a0') : str);
+const _replaceSpace = str => (str?.replace ? str.replace(/\u00A0/g, '\u0020') : str);
```

Comment:
The new code does the inverse of the previous version. I would like this logic to be extracted into a tested unit that can be better understood and reasoned about.

Reply:
I'm unsure if extracting it into a different unit would be the best course of action, seeing as we only use it in one place.